__tests__/app.test.js

```diff
@@ -96,4 +97,79 @@ describe("/api/articles/:article_id", () => {
 
         })
     })
+});
+
+describe("/api/articles", () => {
+    test("GET: 200 - responds with an array of articles without body property", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+            expect(articles).toBeInstanceOf(Array);
```

Comment:
We need a length check here to avoid a potential auto pass if our array of articles is empty